Kitodo/src/main/java/org/kitodo/production/forms/CommentForm.java

```diff
@@ -59,6 +60,19 @@ public Process getProcess() {
         return process;
     }
 
+    /**
+     * Remove a given comment.
+     * 
+     * @param comment to be removed.
+     */
+    public void removeComment(Comment comment) {
+        try {
+            ServiceManager.getCommentService().removeComment(comment);
+        } catch (DAOException e) {
+            e.printStackTrace();
```

Comment:
Use a right logging statement or use the helper method to get the error displayed to the user.